package.json

```diff
@@ -40,7 +40,7 @@
   "dependencies": {
     "@limetech/material-components-web": "^4.0.1-p4.0.0.1",
     "@popperjs/core": "^2.4.2",
-    "@types/tabulator-tables": "^4.6.7",
+    "@types/tabulator-tables": "^4.7.1",
```

Comment:
Hmm… should this really be under `dependencies`, not `devDependencies`? 🤔

@BregenzerK and @jgroth when you're back from vacation (NOT before! 😄)